md/tomcat-bundle.md

```diff
@@ -116,8 +116,9 @@ When you create a tenant, the tenant administrator is created with the default u
 Change these tenant-specific credentials for an existing tenant by updating the `userName` and `userPassword` properties in `bonita-tenant-community-custom.properties`.
```

Comment:
If we decide to keep this information here (instead of bringing or pointing to the "Tenant administrator credentials" section), well then this passage should read:

"Change these tenant-specific credentials for an existing tenant by updating the `userName` and `userPassword` properties in `<WEBSERVER_HOME>/setup/platform_conf/current/tenants/<TENANT_ID>/tenant_engine/bonita-tenant-community-custom.properties`.

Reply:
done